### ISSUE TITLE
I can't view the clipboard anymore after the last update.

### ISSUE THREAD

**Issue by ufkhf** [flathub/com.github.hluk.copyq#47]
I can't view the clipboard anymore after the last update. I can launch CopyQ and see the applet icon in the panel, but when i view the clipboard, CopyQ crashes. I am on the latest Cinnamon DE.

**Comment by hluk** [comment]
What's the app/flatpak version?

**Comment by ufkhf** [comment]
![Copyq](https://user-images.githubusercontent.com/84050866/192239935-4eaaaa3c-887b-4953-9228-005300acfb83.png)

**Comment by hluk** [comment]
What do you mean by "view the clipboard"? Can you post exact steps to reproduce?

Is there any error or warning in the logs? You can view logs with `copyq logs` from terminal.

Please also post the output of `copyq version`.

**Comment by ufkhf** [comment]
At the moment i can see the clipboard this screen:

![Copyq](https://user-images.githubusercontent.com/84050866/192292739-2a252532-31f3-4376-8a1c-8004f0b0c96d.png)

But there are still problems; When i click on the trashcan tab CopyQ crashes. I fixed this by deleting the trashcan tab. Also often when i try to delete an item outside and in the trashcan, CopyQ crashes. 

I have uninstalled CopyQ and it's still the same.
